cyoa/app/views/rounds/marvel_characters_index.html.haml

```diff
@@ -0,0 +1,12 @@
+.row
+  .small-6.large-centered.columns
+    %h2 WHAT IS THE NAME OF THIS CHARACTER?
+    %p
+      .center-image
+        %img{src: @correct_question.trivia_piece}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace